docs/config.rst

```diff
@@ -222,7 +222,7 @@ The following configuration values are used internally by Flask:
 
     If ``session.permanent`` is true, the cookie's expiration will be set this
     number of seconds in the future. Can either be a
-    :class:`datetime.timedelta` or an ``int``.
+    :class:`datetime.timedelta` or a ``float``.
```

Comment:
I'd rather these stay as `int`, accepting a `float` is only a side effect of how `timedelta` parses the value, and it's not really useful here.